src/testdrive/src/main.rs

```diff
@@ -104,8 +104,9 @@ async fn run(args: Args) -> Result<(), Error> {
             }
             (_, aws_endpoint) => {
                 // Either a non-standard AWS region, a custom endpoint, or both. Assume
-                // dummy authentication, and just use default dummy credentials in the
-                // default config.
+                // we have a "custom" region. For us that means 'LocalStack' which
+                // requires an endpoint but the region name itself doesn't matter, and
+                // credentials are ignored.
```

Comment:
Sorry, but I don't follow this new comment. What does "assume we have a 'custom' region" mean? We're not assuming that—we've just been told that! The assumption is that we can use dummy credentials, not that we have a custom region.

Also this block does not intend to know anything about LocalStack and this logic could in theory work with any stubbed out AWS implementation that does not check credentials. I'm not actually sure that the region name doesn't matter in *all* cases; it just so happens that right now we use "localstack" and that seems to work ok. 

What about something like this instead?

> The user specified a non-standard AWS region, a custom endpoint, or both. We instruct Rusoto to use these values by constructing an appropriate `Region::Custom`. We additionally assume we're targeting a stubbed-out AWS implementation that does not check authentication credentials, so we install dummy credentials in the default config.